### DISCUSSION TITLE
Google Analytic4나 Meta Pixel과 같은 외부 스크립트를 삽입할 수도 있나요?

### DISCUSSION THREAD

**Discussion by jay26pluszero** [commerce-api-naver/commerce-solutions#35] [묻고 답하기]
안녕하세요

데이터 수집을 위해 Google Analytics 4나 Meta Pixel과 같은 스크립트를 삽입할 수 있는지 궁금합니다. Shopify의 경우, 사용자 지정 픽셀 설정이 가능한 것으로 알고 있습니다. 만약 외부 스크립트 삽입이 불가능하다면, Shopify Analytics API와 같은 기능이 지원되는지요? 이를 통해 외부 플랫폼에 데이터를 수집할 수 있는지 확인 부탁드립니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

커머스솔루션 연동 중 구체적으로 어느 영역에 외부 스크립트 삽입 가능 여부 확인을 도와드릴 수 있을지 좀 더 자세히 설명해주실 수 있을까요?

**Reply by jay26pluszero** [reply]
네 안녕하세요~!

스마트스토어의 공통 head 영역에 GA4나 GTM, Meta Pixel과 같은 픽셀을 설치할 수 있는지 여부가 궁금합니다. 공식적인 스토어 정책으론 막혀있는 걸로 알고 있습니다. 다만, 솔루션 개발 시에 해당 태그를 심을 수 있는지 궁금합니다.

또한 API데이터솔루션(유료)를 사용하기 위해선 브랜드 스토어가 필수 조건인 걸로 보이는데, 개발사는 어떻게 해당 API를 결제해서 사용할 수 있는지도 궁금합니다!

감사합니다.

**Reply by commerce-api-naver** [reply]
@jay26pluszero 님 안녕하세요. 커머스API 이민석입니다.

### `head` 내 임의 내용 포함은 불가능합니다.
커머스솔루션을 이용한 판매자(사용자) 상품 관리 및 커머스솔루션 정보 관리 과정에서 스마트스토어 혹은 커머스솔루션마켓 내 솔루션 소개 페이지의 `head`로 사용자 정의 내용은 포함할 수 없습니다.

### API데이터솔루션의 커머스솔루션 제공은 현재 지원하지 않습니다.
커머스솔루션의 서비스 제공은 일반적인 스마트스토어/브랜드스토어 판매자의 이용 구조와 다릅니다.
현재 API데이터솔루션은 커머스솔루션 개발사가 이용할 수 있는 구조로 지원하지 않습니다.

**Reply by jay26pluszero** [reply]
답변 감사합니다. 그렇다면, API데이터솔루션(유료)에서 제공되는 고객 데이터, 마케팅 분석, 쇼핑행동 분석, 실시간 분석, 판매 분석 데이터를 활용할 수 있는 방법은 없는 걸까요?

**Reply by commerce-api-naver** [reply]
@jay26pluszero 님, 커머스API 이민석입니다.

현재 API 제공 규격상 커머스솔루션 개발사에는 언급해주신 기능을 제공하지 않습니다.
API데이터솔루션의 현 규격은 판매자(최종 사용자)에게 API 형태의 솔루션을 제공하는 용도로만 설계되어있어 커머스솔루션 개발사의 연동 구조로는 대응이 불가능한 규격입니다.

따라서 현 시점에는 API데이터솔루션에 제공하는 기능을 커머스솔루션 개발사가 솔루션 연동 서비스에 활용할 수 있는 방안이 없음으로 답변드립니다.
